libs/core/src/lib/action-sheet/action-sheet-item/action-sheet-item.component.ts

```diff
@@ -0,0 +1,113 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    EventEmitter,
+    HostListener,
+    Input,
+    Output,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { KeyboardSupportItemInterface } from '../../utils/interfaces/keyboard-support-item.interface';
+import { ButtonComponent } from '../../button/button.component';
+
+
+
+/**
+ * A component used to enforce a certain layout for the action sheet.
+ * ```html
+ * <fd-action-sheet>
+ *     <fd-action-sheet-control>Control Element</fd-action-sheet-control>
+ *     <fd-action-sheet-body>
+ *          <fd-action-sheet-item>Action Sheet Body</fd-action-sheet-item>
+ *          <fd-action-sheet-item>Action Sheet Body</fd-action-sheet-item>
+ *          <fd-action-sheet-item>Action Sheet Body</fd-action-sheet-item>
+ *     </fd-action-sheet-body>
+ * </fd-action-sheet>
+ * ```
+ */
+@Component({
+    selector: 'fd-action-sheet-item',
```

Comment:
Could you make it as a inline component and use it with `li` ( it will be `<li fd-action-sheet-item></>`), just to keep ul>li structure for A11y. Then you would be able to remove `<li>` from body of this component